catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/Context.java

```diff
@@ -30,6 +31,8 @@
 
     private Optional<Metacard> parentMetacard = Optional.empty();
 
+    private final AtomicBoolean isParentDirty = new AtomicBoolean(false);
```

Comment:
Regardless of the term's popularity, I still maintain it would be better to be explicit here as to why the parent metacard is dirty (it needs to be updated with the frame center and location).  This context class has no information on why the parent is dirty and future developers of this code may be at a loss.